jvm/src/main/scala/uk/gov/homeoffice/drt/db/UserFeedbackDao.scala

```diff
@@ -0,0 +1,78 @@
+package uk.gov.homeoffice.drt.db
+
+import akka.stream.scaladsl.Source
+import slick.lifted.{ProvenShape, TableQuery, Tag}
+import slick.jdbc.PostgresProfile.api._
+import uk.gov.homeoffice.drt.feedback.UserFeedback
+
+import scala.concurrent.{ExecutionContext, Future}
+
+case class UserFeedbackRow(email: String,
+                           createdAt: java.sql.Timestamp,
+                           closeBanner: Boolean,
```

Comment:
Sorry I didn't notice this yesterday. The banner being closed doesn't really belong with the feedback, right? The banner can be closed without the user submitting feedback. So it should really be stored separately - could probably be added as a `Timestamp` field in the user table, ie `feedback_banner_closed_at: Timestamp`. I see you're creating an empty feedback row for when the banner is closed at the mo which isn't really right

Reply:
Good point . Putting it with user make sense rather than feedback.